scenarios/demo-s4r-client/app_def.properties

```diff
@@ -0,0 +1,41 @@
+###
```

Comment:
Please conolidate this in `pulsar_workshop/application_deploy/template/app_def.properties.tmpl`. We want to maintain a single application repository.

Also, deployScenario.sh will auto gen `app_def.properties` file under `scenarios/<scenario_name>`, so when you run the script, this file will be over written